extensions/ql-vscode/src/vscode-tests/no-workspace/history-item-label-provider.test.ts

```diff
@@ -140,7 +140,7 @@ describe('HistoryItemLabelProvider', () => {
         const fqi = createMockRemoteQueryInfo({ resultCount: 0, repositoryCount: 2 });
 
         config.format = '%t %q %d %s %f %r %%';
-        expect(labelProvider.getLabel(fqi)).to.eq(`${dateStr} query-name (javascript) 2 repositories in progress query-file.ql  %`);
+        expect(labelProvider.getLabel(fqi)).to.eq(`${dateStr} query-name (javascript) 2 repositories in progress query-file.ql %`);
```

Comment:
Would be nice to have a new test that exercises all sorts of whitespace squashing. After that, looks good.

Reply:
Have added some extra tests.